src/utils/events/move-input.js

```diff
@@ -0,0 +1,32 @@
+const MOVE_EVENTS = ['pointermove', 'touchmove', 'mousemove'];
+
+/**
+ * Hammer.js swallows 'move' events (for pointer/touch/mouse)
+ * when the pointer is not down. This class sets up a handler
+ * specifically for these events to work around this limitation.
+ * Note that this could be extended to more intelligently handle
+ * move events across input types, e.g. storing multiple simultaneous
+ * pointer/touch events, calculating speed/direction, etc.
+ */
+export default class MoveInput {
```

Comment:
Actually, let's hold on combining until we've had a chance for this to land and simmer for a bit. Might turn out we've missed some other things that Hammer doesn't support, or have some other events that need special cases, and there may be a pattern to handle them all together.

Also, I just cleaned up `wheel-input` quite aggressively and would like to keep that separate for now since it has a significant amount of state and logic on its own; will be easier to identify any problems in that refactor if that code stays by itself in its own module.